entrypoint.sh

```diff
@@ -5,6 +5,12 @@ export ALLOWLIST_DIR=/allowlists
 export PYPI_ALLOWLIST="$ALLOWLIST_DIR"/pypi.allowlist
 export CRAN_ALLOWLIST="$ALLOWLIST_DIR"/cran.allowlist
 
+if [ -z "$NEXUS_PATH" ]; then
+    export CONNECT_ARGS="--admin-password $NEXUS_ADMIN_PASSWORD --nexus-host $NEXUS_HOST --nexus-port $NEXUS_PORT"
+else
+    export CONNECT_ARGS="--admin-password $NEXUS_ADMIN_PASSWORD --nexus-host $NEXUS_HOST --nexus-port $NEXUS_PORT --nexus-path $NEXUS_PATH"
+fi
```

Comment:
```suggestion
export NEXUS_PATH=""

export CONNECT_ARGS="--admin-password $NEXUS_ADMIN_PASSWORD --nexus-host $NEXUS_HOST --nexus-port $NEXUS_PORT --nexus-path $NEXUS_PATH"

```

I think we can simplify this a bit, always pass a path just set it to `""` by default.

Reply:
I didn't test this configuration but I avoided it because I was afraid argparse can't interpret an empty value after  a long option. For instance:
` --nexus-path    change-initial-password --path /nexus-data` will probably lead to get  `change-initial-password` as the value of `--nexus-path`

Reply:
Oh, good point. It seemed to work locally but I agree it is a bad idea. I'll try a better way.

Reply:
How about that ☝️

Reply:
Will `export NEXUS_PATH=""` override the environnement variable from the container ? The variable is always empty, isn't it?

Reply:
Yes, good catch! Removed that now.